modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator-to-minor-version_{context}"]
+= Upgrading MetalLB Operator to the next minor version
+
+You can manually upgrade MetalLB Operator to the next minor version by using the {product-title} web console to change the channel of your Operator subscription.
```

Comment:
There's a bit too much in this section.
I'd suggest splitting the content in 2:
- upgrading to the next minor version (assuming we are talking about z streams), where everything should be smooth and straightforward

- upgrading from 4.10, where we should propose the suggested and tested procedure which is:
- remove the 4.10 operator, making sure that the namespace and the metallb cr are not removed
- use the CLI to deploy the 4.11 in the namespace the 4.10 version was